source/hassio/haos_common_tasks.markdown

```diff
@@ -0,0 +1,133 @@
+---
+title: "Home Assistant OS Common Tasks"
+description: "Guides for common tasks when using Home Assistant OS"
+---
+
+This section will provide guides to some common tasks and information which you will need in order to run, maintain, and edit your Home Assistant OS system. For further details on any particular subject, make sure to refer to the documentation for specific add-ons or topics listed here.
+
+## Configuring access to files
+
+One of the first things to take care of after installing Home Assistant OS is to provide yourself access to files. There are several add-ons commonly used for this, and most users employ a mix of various add-ons. Default directories on the host are mapped to the add-ons so that they can be accessed by the services any particular add-on might provide. On the host system these directories exist on the `/data` partition at `/mnt/data/supervisor/`. 
+
+Using any of the add-ons listed below,the following directories are made available for access:
+
+- addons
+- backup
+- config
+- media
+- share
+- ssl
+
+---
+
+### Installing and using the Samba add-on
+
+The Samba add-on creates smb shares which can be accessed from another computer. You can also edit files using the editor of your preference from your client computer.
+
+Configuring the add-on is straight-forward. You only need to set a user and password of your choice on the configuration page, save, and then start the add-on. The add-on will not start without setting a password!
```

Comment:
Avoid using the word `straight-forward`. You're insulting the user in case they cannot figure it out. 

Also mention here that they have to install it from the add-on store.